#!/usr/bin/env python
PACKAGE = "apriltag_tracker"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("max_velocity", double_t, 0, "Maximum Servo Velocity (rad/s)",  0.8, 0.0, 3.0)
gen.add("track_tag", bool_t, 0, "Turn on automatic tag tracking", True)
gen.add("set_position", int_t, 1, "Set the servo to a given position (Only works if track_tag is False)", 512, 0, 1023)

# Third parameter must be equal to file name
exit(gen.generate(PACKAGE, "apriltag_tracker", "DynamicServo"))